environment.go

```diff
@@ -49,3 +76,63 @@ func (e *Environment) dequeueEvent(smID string) (AbstractEvent, bool) {
 	e.queue[smID] = events[1:]
 	return event, true
 }
+
+// SendTo sends an event to a specific state machine.
+// This function must be called from within event handlers registered with
+// OnEvent, OnEntry, OnExit, OnTransition, or OnHalt functions.
+//
+// Parameters:
+//   - ctx: Context passed to the event handler
+//   - target: The state machine that should receive the event
+//   - event: The event to send
+//
+// Example:
+//
+//	OnEntry(spec, IdleState{}, func(ctx context.Context, sm *MyStateMachine) {
+//	    SendTo(ctx, otherSM, Event{Name: "NOTIFY"})
+//	})
```

Comment:
この先の godoc 全部の Example に言えることだけど、
```suggestion
//	goat.OnEntry(spec, IdleState{}, func(ctx context.Context, sm *MyStateMachine) {
//	    goat.SendTo(ctx, otherSM, Event{Name: "NOTIFY"})
//	})
```
と利用者目線で書いてもいいかな